xinterpreters/xinterpreters/host/CMakeLists.txt

```diff
@@ -44,7 +44,10 @@ elseif (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
   target_link_libraries(xtflm_python stdc++ m pthread)
 endif()
 
-include(../cmakefiles/xtflm.cmake)
+set(TOP_DIR
```

Comment:
Let's move CMakeLists.txt to the top directory as it is not specific to host

Reply:
So I'm not so sure about this one, the "project" that CMakeLists builds is specific to host so I think maybe it should stay in that directory?

Reply:
You're right, let's keep it there itself.